tests/test_solution.py

```diff
@@ -225,6 +225,16 @@ def test_evaluate_duplicate_levels_success(self):
 
         assert json == expected_json, f"Expected:\n{expected_json}\nbut got\n{json}"
 
+    def test_death_solutions(self):
```

Comment:
Duplicated above from the other PR; remove

Reply:
Fixed